python/lsst/sims/alertsim/catalogs/diaObjectCommons.py

```diff
@@ -1,179 +1,401 @@
+
+    def get_rLcNonPeriodic(self):
+	vals = self.randomFloatArr(32, -1)
+	cols = ['rLcNonPeriodic01', 'rLcNonPeriodic02', 'rLcNonPeriodic03', 'rLcNonPeriodic04', 'rLcNonPeriodic05', 'rLcNonPeriodic06', 'rLcNonPeriodic07', 'rLcNonPeriodic08', 'rLcNonPeriodic09', 'rLcNonPeriodic10', 'rLcNonPeriodic11', 'rLcNonPeriodic12', 'rLcNonPeriodic13', 'rLcNonPeriodic14', 'rLcNonPeriodic15', 'rLcNonPeriodic16', 'rLcNonPeriodic17', 'rLcNonPeriodic18', 'rLcNonPeriodic19', 'rLcNonPeriodic20']
+	return array_to_dict(cols, vals)
+
+    def get_iLcNonPeriodic(self):
+	vals = self.randomFloatArr(32, -1)
+	cols = ['iLcNonPeriodic01', 'iLcNonPeriodic02', 'iLcNonPeriodic03', 'iLcNonPeriodic04', 'iLcNonPeriodic05', 'iLcNonPeriodic06', 'iLcNonPeriodic07', 'iLcNonPeriodic08', 'iLcNonPeriodic09', 'iLcNonPeriodic10', 'iLcNonPeriodic11', 'iLcNonPeriodic12', 'iLcNonPeriodic13', 'iLcNonPeriodic14', 'iLcNonPeriodic15', 'iLcNonPeriodic16', 'iLcNonPeriodic17', 'iLcNonPeriodic18', 'iLcNonPeriodic19', 'iLcNonPeriodic20']
+	return array_to_dict(cols, vals)
+
+    def get_zLcNonPeriodic(self):
+	vals = self.randomFloatArr(32, -1)
+	cols = ['zLcNonPeriodic01', 'zLcNonPeriodic02', 'zLcNonPeriodic03', 'zLcNonPeriodic04', 'zLcNonPeriodic05', 'zLcNonPeriodic06', 'zLcNonPeriodic07', 'zLcNonPeriodic08', 'zLcNonPeriodic09', 'zLcNonPeriodic10', 'zLcNonPeriodic11', 'zLcNonPeriodic12', 'zLcNonPeriodic13', 'zLcNonPeriodic14', 'zLcNonPeriodic15', 'zLcNonPeriodic16', 'zLcNonPeriodic17', 'zLcNonPeriodic18', 'zLcNonPeriodic19', 'zLcNonPeriodic20']
+	return array_to_dict(cols, vals)
+
+    def get_yLcNonPeriodic(self):
+	vals = self.randomFloatArr(32, -1)
+	cols = ['yLcNonPeriodic01', 'yLcNonPeriodic02', 'yLcNonPeriodic03', 'yLcNonPeriodic04', 'yLcNonPeriodic05', 'yLcNonPeriodic06', 'yLcNonPeriodic07', 'yLcNonPeriodic08', 'yLcNonPeriodic09', 'yLcNonPeriodic10', 'yLcNonPeriodic11', 'yLcNonPeriodic12', 'yLcNonPeriodic13', 'yLcNonPeriodic14', 'yLcNonPeriodic15', 'yLcNonPeriodic16', 'yLcNonPeriodic17', 'yLcNonPeriodic18', 'yLcNonPeriodic19', 'yLcNonPeriodic20']
+	return array_to_dict(cols, vals)
+    
+    def get_nearbyObj1(self):
+	return self.randomInts(-1, 9223372036854775807)
+    
+    def get_nearbyObj1Dist(self):
+	return self.randomFloats(-1)
+    
+    def get_nearbyObj1LnP(self):
+	return self.randomFloats(-1)
+    
+    def get_nearbyObj2(self):
+	return self.randomInts(-1, 9223372036854775807)
+    
+    def get_nearbyObj2Dist(self):
+	return self.randomFloats(-1)
+    
+    def get_nearbyObj2LnP(self):
+	return self.randomFloats(-1)
+    
+    def get_nearbyObj3(self):
+	return self.randomInts(-1, 9223372036854775807)
+    
+    def get_nearbyObj3Dist(self):
+	return self.randomFloats(-1)
+    
+    def get_nearbyObj3LnP(self):
+	return self.randomFloats(-1)
+    
+    # flags
+
+
+##################
+
+
+
+    # resolve db column case-sensitivness
+    #def get_htmId20(self):
+    #    return self._decapitalize_column_name('htmID')
+
+    def _decapitalize_column_name(self, colname):
```

Comment:
It does not look like this method is used.  It should be removed.

If I am wrong, you should have it raise an exception if neither `colname` nor `colname.lower()` are valid columns.

Reply:
Will remove. This was a workaround at some point.

Reply:
also removed from diaSourceCommons.py